<?xml version="1.0"?>

<launch>

	<!--Launch the camera-->
	<include file="$(find mocapcam)/launch/start_cam.launch" />

	<!--Run the aruco detection-->
	<node pkg="mocapcam" name="detection" type="detect.py" args="" output="screen">
		<param name="box_size" type="double" value="18.0" />
		<param name="aruco_size" type="double" value="13.0" />
		<param name="image_topic" type="string" value="/usb_cam/" /> <!-- camera_info and image_raw -->
		<param name="pose_topic" type="string" value="box_pose" />

		<rosparam command="load" file="$(find mocapcam)/config/transforms.yaml" />
	</node>

	<!--Run rviz-->
	<node pkg="rviz" name="visualization" type="rviz" args="-d $(find mocapcam)/rviz/config.rviz" output="screen">
	</node>

</launch>